lib/activerecord/delay_touching.rb

```diff
@@ -6,9 +6,9 @@ module DelayTouching
     extend ActiveSupport::Concern
 
     # Override ActiveRecord::Base#touch.
-    def touch(name = nil)
+    def touch(*names)
       if self.class.delay_touching? && !try(:no_touching?)
-        DelayTouching.add_record(self, name)
+        DelayTouching.add_record(self, names)
```

Comment:
perhaps adding a raise here if names.flatten.compact.length == 0. If I read the earlier code correctly, it would have raised a ArgumentError at this point since add_record requires name to be a defined value. 
If names.flatten.compact is passed down to add_record, there would be no need to normalize_columns.